include/deal.II/matrix_free/fe_evaluation_data.h

```diff
@@ -516,7 +530,7 @@ class FEEvaluationData
                       internal::MatrixFreeFunctions::DoFInfo::dof_access_cell)
       return cell_ids;
     else
-      return cell_or_face_ids;
+      return face_ids;
```

Comment:
... and modified the return value here. As far as I see it, this should be fine.